src/Pages/Stay/StayMain/StayMain.js

```diff
@@ -0,0 +1,323 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import StayMainDiscount from './Components/StayMainDiscount';
+import StayMainRecommend from './Components/StayMainRecommend';
+import StayMainSearchCategory from './Components/StayMainSearchCategory';
+import StayCalendar from './Modal/Calendar/StayCalendar';
+
+const StayMain = () => {
+  const [adult, setAdult] = useState(1);
+  const [room, setRoom] = useState(1);
+  const [isUserBoxToggle, setIsUserBoxToggle] = useState(false);
+
+  const handleAdultAdd = () => {
+    if (adult > 8) {
+      alert('최대 정원 9명까지만 가능합니다.');
+    } else {
+      setAdult(adult + 1);
+    }
+  };
```

Comment:
삼항연산자로 구현할 수 있겠네요! :)
다른 곳에서도 이와 같은 로직을 적용해주세요!
```suggestion
  const handleAdultAdd = () => {
    adult > 8 ? alert('안내 메세지') : setAdult(adult + 1);
  };
```

Reply:
넵 다 확인 후 수정했습니다 감사합니다 ~